src/pybind/rbd/rbd.pyx

```diff
@@ -4459,10 +4459,87 @@ written." % (self.name, ret, length))
         """
         List image-level config overrides.
 
-        :returns: :class:`ConfigPoolIterator`
+        :returns: :class:`ConfigImageIterator`
         """
         return ConfigImageIterator(self)
 
+
+    def config_image_set(self, key, value):
```

Comment:
how about `image_config_set`?

Reply:
hi，tchaikov，“rbd config image set” in CLI，so I use config_image_set, thanks.

Reply:
i see, but i was trying to align with other namings in the same source file. for instance, `mirror_image_get_status` and `snap_get_group_namespace`. so probably `image_config_set` is better in terms of readability and consistency?

Reply:
ok, I will use  image_config_set and align with other namings, thanks

Reply:
Actually, now that you point it out, the `image` prefix can just be dropped since it's already a method in the `Image` class.

Reply:
👍 

and "set_config" might be a better name than "config_set".

Reply:
hi，tchaikov，I feel “config_set” is better, because there is a "config_list" method in Image class. It should align with other namings in the same class, thanks.

Reply:
I am also fine w/ the (noun)_(verb) style since it’s used throughout.

Reply:
sure, guess i just forgot that we discussed the (none)_(verb) naming before.

Reply:
I have used noun)_(verb) style，thanks a lot.